recipe/meta.yaml

```diff
@@ -13,8 +13,7 @@ source:
       - pkg-config-do-not-add-requires-private.patch
 
 build:
-  number: 0
-  skip: true  # [win]
+  number: 1
```

Comment:
no need to bump, this doesnt change existing unix packages

Reply:
Fixed in https://github.com/conda-forge/ipopt-feedstock/pull/67/commits/1bc668f8da1331b3940a083756b8eca77b439432 .